### ISSUE TITLE
[v2] Can't remove topBar subtitle (and re-align topBar title)

### ISSUE THREAD

**Issue by deecewan** [wix/react-native-navigation#4374]
### Issue Description

When you've set a subtitle, there is no way to remove the subtitle, and have the title go back to full size.

### Steps to Reproduce / Code Snippets / Screenshots

A full size title:
![image](https://user-images.githubusercontent.com/4755785/49264071-a8dc0d00-f498-11e8-9135-a16376481204.png)

A title with a sub title:
![image](https://user-images.githubusercontent.com/4755785/49264087-b42f3880-f498-11e8-823e-3fb855b3282a.png)

A title that had a sub title, and it's now been removed (note the size of the title, and the space between title and edge of bar):
![image](https://user-images.githubusercontent.com/4755785/49264108-c4dfae80-f498-11e8-9104-715b29ff86b2.png)

I believe you can reproduce this by doing the following:

```js
Navigation.mergeOptions(this.props.componentId, {
  topBar: {
    title: {
      text: 'My Title',
    },
    subtitle: {
      text: 'My Subtitle',
    },
  }
});

Navigation.mergeOptions(this.props.componentId, {
  topBar: {
    title: {
      text: 'My Title',
    },
    subtitle: {
      text: '',
    },
  }
});
```

I have tried setting the whole subtitle object to `undefined` (it just remains on the screen regardless), `null` (it breaks and complains that null is not a valid option) and also setting the `subtitle.text` option to `undefined` and `null` with the same results (stays visible, or crashes the app)

---
### Environment
* React Native Navigation version: 2.1.3
* React Native version: 0.57.3
* Platform(s) (iOS, Android, or both?): iOS (maybe both - haven't checked android)
* Device info (Simulator/Device? OS version? Debug/Release?): Simulator, iPhone X, iOS 12

**Comment by henrikra** [comment]
@guyca I noticed the same thing that there is no way to remove subtitle once it is added. We have faced same issues that we would like to remove subtitle at specific point. Can you guys check this out?

**Comment by deecewan** [comment]
This is definitely not stale. It just hasn't been looked at by anyone.

**Comment by henry132109** [comment]
Just dropping by to say that the bug is still there.

**Comment by danilobuerger** [comment]
Fixed by #5883 (auto close in PR didn't work)
